spring-cloud-gcp-samples/pom.xml

```diff
@@ -33,6 +33,7 @@
 		<module>spring-cloud-gcp-pubsub-binder-sample</module>
 		<module>spring-cloud-gcp-integration-pubsub-json-sample</module>
 		<module>spring-cloud-gcp-security-iap-sample</module>
+		<module>spring-cloud-gcp-sql-postgres-sample</module>
```

Comment:
Should we also rename `spring-cloud-gcp-sql-sample` to `spring-cloud-gcp-sql-mysql-sample`?